modules/Settings/PickListDependency/views/List.php

```diff
@@ -24,12 +24,10 @@ public function process(\App\Request $request)
 	{
 		if ($request->isAjax()) {
 			$moduleModelList = Settings_PickListDependency_Module_Model::getPicklistSupportedModules();
-			$forModule = $request->getByType('formodule', 'Alnum');
-
+			$forModule = $request->get('formodule');
```

Comment:
We recomend use function isEmpty

$forModule = '';
if (!$request->isEmpty('formodule')) {
			$forModule = $request->getByType('formodule', 'Alnum');
}

Reply:
done